top-k-frequent-elements/yolophg.js

```diff
@@ -0,0 +1,28 @@
+// Time Complexity: O(n)
+// Space Complexity: O(n)
+
+var topKFrequent = function(nums, k) {
+    const frequencyMap = new Map();
+    // for each number in the array, update frequency.
+    for (const num of nums) {
+        frequencyMap.set(num, (frequencyMap.get(num) || 0));
```

Comment:
1을 더해주지 않으면, 항상 0이 되지 않나요? 😳

```suggestion
        frequencyMap.set(num, (frequencyMap.get(num) || 0) + 1);
```

Reply:
헉, 놓친 부분을 지적해주셔서 감사합니다! 수정하여 반영하였습니다.